src/configdialog.cpp

```diff
@@ -394,6 +394,10 @@ ConfigDialog::ConfigDialog(QWidget *parent): QDialog(parent,Qt::Dialog|Qt::Windo
     // adapt icon size to dpi
     double dpi=QGuiApplication::primaryScreen()->logicalDotsPerInch();
     double scale=dpi/96;
+	int systemdpi = qRound(QGuiApplication::primaryScreen()->physicalDotsPerInch()); // main screen dpi
+	if (systemdpi < 10 || dpi > 1000)
+		systemdpi = 96;
+	QString labelSystemdpi = tr("Screen Resolution") + " (" + tr("System") + QString(": %1 dpi").arg(systemdpi) + "):";
```

Comment:
tr("%1").arg(12) works, so please change the text to a single tr expression.

Reply:
thx